tests/conftest.py

```diff
@@ -31,6 +31,7 @@ def database_name_generator():
             count=i,
         )
 
+
```

Comment:
`tests/conftest.py:34:1: E305 expected 2 blank lines after class or function definition, found 1` but I'm reverting this.